### PR TITLE
[SELC-5871] feat: create WorkflowExecutorForUsersEa to manage different templates for user aggregators

### PR DESCRIPTION
**Pull request by giulia-tremolada**
#### List of Changes
-  create WorkflowExecutorForUsersEa to manage different templates for user aggregators
- modify PdfModify to manage new variables in create users for aggregators template

#### Motivation and Context
This change is required to use a different template when adding users with signature for aggregators

#### How Has This Been Tested?
It was tested locally

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/onboarding-functions/src/main/java/it/pagopa/selfcare/onboarding/utils/UserUtils.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import it.pagopa.selfcare.onboarding.entity.Onboarding;
+import it.pagopa.selfcare.onboarding.entity.User;
+
+import java.util.Objects;
+
+public class UserUtils {
+
+  public UserUtils() {}
+
+  public static String getEmailRegistrationTemplatePath(MailTemplatePathConfig config, Onboarding onboarding) {
```
Comment (manuraf) [inline apps/onboarding-functions/src/main/java/it/pagopa/selfcare/onboarding/utils/UserUtils.java:14]: you could Create unit tests for UserUtils.getEmailRegistrationTemplatePath to verify its behavior under various scenarios, including edge cases....in addition, you could add some comments to the UserUtils method to explain its purpose, parameters, and return values for future maintainers.

**Review by manuraf — APPROVED** [review]
